samples/DEMO.md

```diff
@@ -921,6 +951,75 @@ hts_idx_destroy. The hts_reglist_t* array passed is destroyed by the library
 on iterator destroy. The regions array (array of char array/string) needs to be
 destroyed by the user itself.
 
+For fasta/fastq files, the index has to be loaded using fai_load3_format which
+takes the file, index file names and format. With single region specification
+fai_fetch64 can be used to get bases, and fai_fetchqual64 for quality in case
+of fastq data. With multiple region specification, with comma separattion,
+faidx_fetch_seq64 and faidx_fetch_qual64 does the job. Regions has to be parsed
+using fai_parse_region in case of multiregion specifications. fai_adjust_region
+is used to adjust the start-end points based on available data.
+
+Below excerpt shows fasta/q access with single and multiregions,
+
+    ...
+    //load index
+    if (!(idx = fai_load3_format(inname, NULL, NULL, FAI_CREATE, fmt))) {
+    ...
+        //get data from given region
+        if (!(data = fai_fetch64(idx, region, &len))) {
+    ...
+        else {
+            printf("Data: %"PRId64" %s\n", len, data);
+            free((void*)data);
```

Comment:
yes, it is just handling the error and it is masked in '...' to avoid cluttering and to show only relevant API and comments.
As the user has to check the .c file for details, it will be visible.